src/module_utils.hpp

```diff
@@ -8,21 +9,14 @@ namespace utils {
 * throwing errors.
 * Usage:
 *
-* v8::Local<v8::Function> callback;
-* return CallbackError("error message", callback);  // "return" is important to
-* prevent duplicate callbacks from being fired!
-*
-*
-* "inline" is important here as well. See for more contex:
-* - https://github.com/mapbox/cpp/blob/master/glossary.md#inline-keyword
-* - https://github.com/mapbox/node-cpp-skel/pull/52#discussion_r126847394 for
-* context
-*
+* Napi::Function callback;
+* return CallbackError("error message", callback);
 */
-inline void CallbackError(std::string message, v8::Local<v8::Function> func)
+
+inline Napi::Value CallbackError(std::string const& message, Napi::CallbackInfo const& info, Napi::Function const& func)
```

Comment:
Curious question: do the https://nodejs.org/api/n-api.html#n_api_n_api_callback_types types come into play here?